extensions/sql-bindings/src/test/common/azureFunctionsUtils.test.ts

```diff
@@ -0,0 +1,103 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as vscode from 'vscode';
+import * as path from 'path';
+import * as os from 'os';
+import * as uuid from 'uuid';
+import * as fs from 'fs';
+import * as rimraf from 'rimraf';
+import * as should from 'should';
+import * as sinon from 'sinon';
+import { promisify } from 'util';
+import * as constants from '../../common/constants';
+import * as azureFunctionsUtils from '../../common/azureFunctionsUtils';
+import { assert } from 'console';
+
+let rootFolderPath: string;
+let localSettingsPath: string;
+let projectFilePath: string;
+
+describe('Tests to verify Azure Functions Utils functions', function (): void {
+	beforeEach(async () => {
+		rootFolderPath = path.join(os.tmpdir(), `AzureFunctionTest_${uuid.v4()}`);
+		await fs.mkdirSync(rootFolderPath);
```

Comment:
Unit tests should generally not be making actual files. Use sinon to stub these out instead and return fake results.